hack/debug.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+root=$(dirname "${BASH_SOURCE[0]}")
```

Comment:
I just tried this locally is it always returns `.`. Is this expected and it it is always `.` do we need it declared this way?

Reply:
This makes it sort of independent of the `pwd` from which the script was invoked.

Reply:
I see, ok, my idea was that if the value is always the current directory `./`, why don't we just use the literal `./`.